### PR TITLE
JBIDE-26296 Remove JBoss string

### PR DESCRIPTION
**Pull request by mmalina**
This was missed in a bulk change in JBDS-4709

### CODE REVIEW

**Comment by fbricon** [comment]
@mmalina can you please sign off your commit?

**Comment by mmalina** [comment]
@fbricon damn, I did this simple edit just using github web and didn't realize this. I guess I'll have to do it locally anyway :) wait a sec

**Comment by mmalina** [comment]
@fbricon done

File: org.jboss.tools.maven.apt.feature/feature.properties [hunk 1]
```diff
@@ -37,7 +37,7 @@ http://www.eclipse.org/legal/epl-v10.html\n\
 \n\
 Contributors:\n\
      Knowledge Computing Corp. - initial API and implementation\n
-     JBoss by Red Hat, Inc. - Packaging\n
+     Red Hat                   - Packaging\n
```
Comment (nickboldt) [inline org.jboss.tools.maven.apt.feature/feature.properties]: This should probably be Red Hat, Inc.

File: org.jboss.tools.maven.apt.feature/feature.properties [hunk 2]
```diff
@@ -7,7 +7,7 @@
 # 
 # Contributors:
 #     Knowledge Computing Corp. - initial API and implementation\n
-#     JBoss by Red Hat          - Packaging\n
+#     Red Hat                   - Packaging\n
```
Comment (nickboldt) [inline org.jboss.tools.maven.apt.feature/feature.properties]: This should probably be Red Hat, Inc.

File: org.jboss.tools.maven.apt.feature/feature.properties (hunk 1 above)
Comment (mmalina) [reply org.jboss.tools.maven.apt.feature/feature.properties]: @nickboldt it looks identical to line 10, so why should it be different?

Comment (mmalina) [reply org.jboss.tools.maven.apt.feature/feature.properties]: @nickboldt I can put it back there easily if you think that's required. But there's no Inc. anywhere here for example:
https://github.com/jbosstools/jbosstools-central/pull/434/files

**Pull request merged**